### ISSUE TITLE
Give user permission to collaborate on organisation snaps

### ISSUE THREAD

**Issue by nottrobin** [canonical-ols/build.snapcraft.io#622]
So that the user doesn't need to be directly signed in to the organisation's SSO account, either alongside or instead of their existing SSO account, we should add collaboration information to the store so the user has direct access to build and publish the snap.

This cannot start until changes have been made to the snap store to support collaboration.

More details can be found in [this discussion](https://forum.snapcraft.io/t/github-organisations-proposal/157).

**Comment by nottrobin** [comment]
No longer relevant.
